lib/LMSManagers/LMSNetworkManager.php

```diff
@@ -879,4 +881,53 @@ public function getFirstFreeAddress($netid)
         }
         return $ip;
     }
+
+    public function GetVlanList()
+    {
+        return $this->db->GetAll(
+            'SELECT vl.id, vlanid, description, customerid, CONCAT(cv.lastname,\' \',cv.name) AS customername
```

Comment:
w PGSQL nie zadziala. Użyj LMSDB::Concat().

Reply:
@chilek czy przez PGSQL rozumiesz postgresql tam jak najbardziej działa - np. https://w3resource.com/PostgreSQL/concat-function.php 

a w UI widać to tak (instancja testowa z bazą postgres):
![działa-concat](https://user-images.githubusercontent.com/17087236/98346121-d2ebc180-2015-11eb-98f7-72dd7542fd93.png)

Reply:
Patrzysz w knigu widzisz figu :)

Wymagane: PostgreSQL Version : 9.3

a w CentOS 7 mamy 9.2.

Proponuję nie zwiększać wymagań z głupiego powodu.